### PR TITLE
Mobile fun

### PR DESCRIPTION
**Pull request by nhunzaker**
respond
calll

### CODE REVIEW

File: src/style/components/search.scss [hunk 1]
```diff
@@ -2,8 +2,8 @@
   background: #fff;
   border-radius: 2px;
   box-shadow: 0 1px 2px rgba(#000, 0.4);
-  padding: 8px 16px;
-  margin: 8px;
+  padding: 8px 0;
```
Comment (solomonhawk) [inline src/style/components/search.scss]: :abc:

File: src/style/components/gallery.scss [hunk 2]
```diff
@@ -1,10 +1,23 @@
 $ars-gallery-height: 432px;
 
 .ars-gallery {
+  display: flex;
+  flex-grow: 1;
+  flex-wrap: wrap;
   height: $ars-gallery-height;
-  padding: 8px 4px;
-  margin: 0 auto;
+  padding: 6px 4px 2px;
```
Comment (solomonhawk) [inline src/style/components/gallery.scss]: :abc:

**Comment by solomonhawk** [comment]
This looks great, no real feedback. :+1:

File: src/style/components/search.scss (hunk 1 above)
Comment (nhunzaker) [inline src/style/components/search.scss]: :cry:

**Comment by greypants** [comment]
:+1:

**Comment by tommymarshall** [comment]
You could transition a `max-height` so the search results would transition up into place instead of snapping to the top. Maybe...

:+1:

**Comment by nhunzaker** [comment]
@tommymarshall generally, I'm not happy with the transition as search results fade out. I definitely want to tinker with it more.

**Pull request merged**